runtime/integration-tests/src/evm/precompile.rs

```diff
@@ -91,8 +91,14 @@ async fn axelar_precompile_execute() {
 
 	let currency_metadata = AssetMetadata {
 		decimals: 18,
-		name: "Test".into(),
-		symbol: "TST".into(),
+		name: BoundedVec::<u8, <Runtime as pallet_pool_system::Config>::StringLimit>::try_from(
+			"Test".as_bytes().to_vec(),
+		)
+		.expect("Can create BoundedVec for token name"),
+		symbol: BoundedVec::<u8, <Runtime as pallet_pool_system::Config>::StringLimit>::try_from(
+			"TST".as_bytes().to_vec(),
+		)
+		.expect("Can create BoundedVec for token symbol"),
```

Comment:
I think just `BoundedVec::default()` is enough for this test. To simplify a bit this.

```suggestion
		name: BoundedVec::default(),
		symbol: BoundedVec::default(),
```

Reply:
Hmm, let's try this, even tho it might make life a bit harder when debugging.